spec/eventbrite_sdk/event_spec.rb

```diff
@@ -197,6 +197,18 @@ module EventbriteSDK
             url: 'events/1/cancel'
           )
         end
+
+        it 'when given :api_token it passes it through' do
```

Comment:
Would it be overkill to test this in `publish` and `unpublish` as well since they're both defined through `define_api_actions`?